shell/edit/monitoring.coreos.com.alertmanagerconfig/auth.vue

```diff
@@ -252,9 +248,9 @@ export default {
         />
       </div>
     </div>
-    <div
-      v-if="authType === 'basicAuth'"
-      class="row mb-20"
+    <div
+      v-if="authType === 'basicAuth'"
+      class="row mb-20"

```

Comment:
What's going on in this file? I don't see any differences. It looks like even the spacing is the same.

Why is there a diff here? I must be blind.

Reply:
I should have noted in my PR description to review this file commit-by-commit. We replace `$set` with assignment in `shell/edit/monitoring.coreos.com.alertmanagerconfig/auth.vue` 

https://github.com/rancher/dashboard/pull/12253/commits/cf1341cd44f939b4f7fc0eab4bae23ab24d75348#diff-c78b91f005a7b21458691f634c58247ac83dfde2480a9d21b36bace278da2b2e

And in a separate commit, we apply the auto-fixes.

Reply:
Actually, I might understand your question now that I think about it a little more. I think this might have something to do with line endings for this file, so it must not be a linter change afterall. 

For some reason `shell/edit/monitoring.coreos.com.alertmanagerconfig/auth.vue` is reporting CRLF. Do you see the same?

Reply:
I think that we can omit 13ea8af10ab273d442c0e18f8854c0be67193c9f from this PR and deal with line endings in a different PR if we feel that this needs to be addressed.